src/exercises/3-Basic_Primitives/Readme.md

```diff
@@ -1,10 +1,10 @@
-In this exercise we’ll create reusable primitive components. We’ll create a text and a heading component. We’ll learn how to reduce boilerplate by using [styled-system](https://styled-system.com).
+In this exercise we’ll create reusable primitive components. We’ll create a Text, Link, and a Heading components. We’ll learn how to reduce boilerplate by using [styled-system](https://styled-system.com) and how to compose primitives.
 
 The majority of every user interface is a text. Unsurprisingly, most of the inconsistencies are coming from text styles. To reduce the amount of different text styles we should restrict our styling and only allow using design tokens.
 
 ## 3.1. Creating a generic text component
 
-In the previous exercise we learned how to use design tokens instead of arbitrary CSS values in our components. Now we need a way to write styles and access design tokens inside our application code. I.e. we want to design a component that implements an API to our design system and doesn’t require writing CSS.
+In the previous exercise we learned how to use design tokens instead of arbitrary CSS values in our components. Now we need a way to write styles and access design tokens inside our application code. without writing custom CSS.
```

Comment:
```suggestion
In the previous exercise we learned how to use design tokens instead of arbitrary CSS values in our components. Now we need a way to write styles and access design tokens inside our application code without writing custom CSS.
```